app.js

```diff
@@ -82,33 +52,47 @@ centerImage.src = allImages[centerImageIndex].filePath;
 
 
 
-// Helpers and Handlers
+
+//==============================================================//
+// Handlers
+//==============================================================//
+
 function handleClickOnImage (clickEvent){
   if(event.target.tagName !== 'IMG'){
     console.log('click on an image');
     return;
   }
 
   totalClicks++;
-  //figure out which image was clicked
-  // This broke if i didnt click on an image
-  // Do this for all images
-
+  
   for( var i = 0; i < allImages.length; i++){
     if(event.target.src.includes(allImages[i].filePath)){
       console.log(`${allImages[i].name} was picked`);
       allImages[i].clickCount++;
     }
   }
   pickThreeImagesAndIncrementAppeared();
-  // check if i have clicked 5 times, then put the chart on
   if(totalClicks > 25){
-    buildMyChart();
+   buildMyChart();
+   removeImage();
+   
+   var stringifyallImages = JSON.stringify(totalClicks);
+   localStorage.setItem('imageInfo', stringifyallImages);
```

Comment:
this is properly storing all the clicks in local storage, 
For the lab requirement you need to store your array of busmall image objects